### PR TITLE
changed create buttons to be more visible

### PR DESCRIPTION
**Pull request by KatBirk**

### CODE REVIEW

File: resources/views/rolesAndPermissions/roles.blade.php [hunk 1]
```diff
@@ -4,12 +4,19 @@
     @include('partials._systemFeedback')
 
     <div class="container">
-        <h2>{{__('customLabels.roles')}}</h2>
-        <div class="d-grid d-md-flex justify-content-md-end"> <br>
-            <a class="btn btn-outline-primary mb-2" role="button" href="{{route('admin.roles.create')}}">{{__('customLabels.create')}}</a>
+        <h2 class="text-center mb-5">{{__('customLabels.roles')}}</h2>
```
Comment (Darchie4) [inline resources/views/rolesAndPermissions/roles.blade.php:7]: Use of deprecated labels

File: resources/views/rolesAndPermissions/roles.blade.php [hunk 2]
```diff
@@ -24,8 +31,10 @@
                             <td>{{$role->id}}</td>
                             <td>{{$role->name}}</td>
                             <td>
-                                <a role="button" class="btn btn-outline-primary" href="{{route('admin.roles.edit', $role->id)}}">{{__('customLabels.edit')}}</a>
-                                <form class="d-inline-flex" method="post" action="{{route('admin.roles.destroy',$role->id)}}"
+                                <a role="button" class="btn btn-outline-primary"
+                                   href="{{route('admin.roles.edit', $role->id)}}">{{__('customLabels.edit')}}</a>
```
Comment (Darchie4) [inline resources/views/rolesAndPermissions/roles.blade.php:35]: use of deprecated labels

**Review by Darchie4 — CHANGES_REQUESTED** [review]
Please don't use the deperated labels

**Comment by KatBirk** [comment]
You can't tell me what to do

**Pull request merged**